tests/avatar/test_service.py

```diff
@@ -0,0 +1,630 @@
+########################################################################
+# File name: test_service.py
+# This file is part of: aioxmpp
+#
+# LICENSE
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+########################################################################
+import base64
+import binascii
+import hashlib
+import unittest
+
+import aioxmpp
+import aioxmpp.errors as errors
+import aioxmpp.xso.model as xso
```

Comment:
Please do not import ``aioxmpp.xso.model`` directly, use ``aioxmpp.xso``. It should export everything you need.